src/T2DMap.cpp

```diff
@@ -1721,7 +1721,7 @@ void T2DMap::paintEvent( QPaintEvent * e )
                 _pt = QPointF( rx-(tx*rSize)/3.1, ry+(ty*rSize)*uDirection.z()/3.1 );
                 _poly.append( _pt );
                 QBrush brush = p.brush();
-                brush.setColor( QColor(0, 0 ,0) );
+                brush.setColor( QColor(Qt::black) );
```

Comment:
In some places I think you can drop the constant in without a `QColor(`...`)` around the outside - especially if a colour is expected (as a `const` reference perhaps?)